pkg/api/volopt_level.go

```diff
@@ -0,0 +1,29 @@
+package api
```

Comment:
This must be defined on original type at `glusterd2/xlator/options/options.go`

Reply:
removed this file